### DISCUSSION TITLE
how to use http post ardio data

### DISCUSSION THREAD

**Discussion by weirdorr** [pschatzmann/arduino-audio-tools#1564] [Q&A] [answered]
### Problem Description

how to use http post ardio data

### Device Description

how to use http post ardio data

### Sketch

```cpp
how to use http post ardio data
```

### I have checked existing issues, discussions and online documentation

- [X] I confirm I have checked existing issues, discussions and online documentation

**Comment by pschatzmann** [comment, marked as answer]
I dont see any bug here, so I move it to a discussion. 
You are lacking some critical details: which platform is doing the post and which one is supposed to process it ?

The URLStream class can post data if the post should be done from Arduino!

**Reply by weirdorr** [reply]
i want sen data to my url

**Reply by weirdorr** [reply]
send I2S microphone data to a remote HTTP endpoint using POST

**Reply by pschatzmann** [reply]
So, I already gave you the answer
